client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -125,6 +139,23 @@ const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId, transl
 		},
 	} );
 
+	const getExceedQuotaErrorContent = () => {
+		return (
+			<ExceedQuotaErrorWrapper data-testid="quota-message">
+				<Notice status="is-warning" showDismiss={ false }>
+					<p>
+						{ __(
+							'It appears that your available size quota is currently insufficient for creating a staging site.'
```

Comment:
Maybe we could say "Your available storage space is lower than 50%, which is insufficient for creating a staging site, " so the user will know the real cause. We don't need to invite them to contact support here too.